### ISSUE TITLE
Generate requirements in meta.yaml from setup.py?

### ISSUE THREAD

**Issue by jbiggsets** [EducationalTestingService/rsmtool#297]
Right now, we're explicitly listing out all of the requirements in the `meta.yaml` file:

https://github.com/EducationalTestingService/rsmtool/blob/f5b63fa4324a7c4cdf4e382f085348943cec15ad/conda-recipe/rsmtool/meta.yaml#L22-L46

Some of these (like `sphinx`) we have to list out explicitly, but the rest we could probably populate directly from `setup.py` file, so we don't have to update both of these things and make sure they're consistent with each release.

I believe we can use this approach instead:

```
{% set data = load_setup_py_data() %}

...

{% block requirements -%}
requirements:
  build:
    {% for req in data.get('install_requires', []) -%}
    - {{(req.replace(">", " >") if ">" in req else req.replace("==", " "))}}
    {% endfor %}
  run:
    - python
    {% for req in data.get('install_requires', []) -%}
    - {{(req.replace(">", " >") if ">" in req else req.replace("==", " "))}}
    {% endfor %}
{%- endblock %}
```

Thoughts?

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#id15

**Comment by desilinguist** [comment]
Ooh, this looks like a great idea! Have you already tested it and it works?

**Comment by jbiggsets** [comment]
Testing it out now. 👍

**Comment by desilinguist** [comment]
Addressed by #305
